docs/Users_Guide/tcrmw_cross_section.rst

```diff
@@ -11,7 +12,7 @@ https://metcalcpy.readthedocs.io/en/develop/Users_Guide/vertical_interpolation.h
 
 
 Example
-~~~~~~~
+=======
 
 
 **Sample Data**
```

Comment:
```suggestion
Sample Data
-------------
```


Hi @lisagoodrich.  According to the style guide here, https://github.com/dtcenter/METplus/blob/feature_1667_updating_overview/docs/Contributors_Guide/documentation.rst
it looks like "**Sample Data**" (and "**Configuration Files**") later in the file should not be bolded. but should have a dash underneath:
```
- for subsections (this is a dash, not an underline.)
```
I was going to say that it should look like this page in the METplotpy repo for "**Sample Data**" and "**Configuration Files**" but it looks like this page has underscores and the not dashes it should have:
https://raw.githubusercontent.com/dtcenter/METplotpy/develop/docs/Users_Guide/bar.rst

Perhaps you could take another look at the other pages to change underscores to dashes and check for other instances that need improvements?  Thanks!

Reply:
@jprestop thank you for catching these!  I didn't know to check for bolding vs headers.  

And thanks for the dashes vs underscores.  This means many more files need updating.  It can be hard for me to see the difference between '_' and "-" when it's being used as an underline.  I'll review the documentation again.  Thanks!

Reply:
> It can be hard for me to see the difference between '_' and "-" when it's being used as an underline. I'll review the documentation again.
I totally agree @lisagoodrich.  I usually highlight the line and then it becomes obvious, otherwise, I have a hard time telling too.

Reply:
@jprestop, Thanks, I'll re-review ALL of the pages and look for this specifically.